frontend/src/pages/analytics/comps/ExploreEmbedFrame.tsx

```diff
@@ -10,67 +10,61 @@
  *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
  *  and limitations under the License.
  */
-import { Box } from '@cloudscape-design/components';
 import { createEmbeddingContext } from 'amazon-quicksight-embedding-sdk';
 import React, { useEffect } from 'react';
-import { useTranslation } from 'react-i18next';
 
 interface ExploreEmbedFrameProps {
-  embedType: 'dashboard' | 'visual';
+  embedType: 'dashboard' | 'visual' | 'console';
   embedUrl: string;
-  embedId: string;
 }
 
 const ExploreEmbedFrame: React.FC<ExploreEmbedFrameProps> = (
   props: ExploreEmbedFrameProps
 ) => {
-  const { t } = useTranslation();
-  const { embedType, embedUrl, embedId } = props;
+  const { embedType, embedUrl } = props;
 
   const embedContainer = async () => {
     const embeddingContext = await createEmbeddingContext();
     switch (embedType) {
       case 'dashboard':
         await embeddingContext.embedDashboard({
           url: embedUrl,
-          container: `#${embedId}`,
+          container: `#EmbedId`,
           resizeHeightOnSizeChangedEvent: true,
         });
         break;
       case 'visual':
         await embeddingContext.embedVisual({
           url: embedUrl,
-          container: `#${embedId}`,
+          container: `#EmbedId`,
           resizeHeightOnSizeChangedEvent: true,
         });
         break;
+      case 'console':
+        await embeddingContext.embedConsole(
+          {
+            url: embedUrl,
+            container: `#EmbedId`,
+            resizeHeightOnSizeChangedEvent: true,
+            height: '1000',
+          },
+          {
+            locale: 'en-US',
```

Comment:
use the solution's language instead?

Reply:
solution's language save in localStorage that is 'en', Quicksight's locale is 'en-US'.

Reply:
> @chenhaiyun, could we specify en-US and then fallback to en in the front end?

Do you mean we save the solutions's default language to localStorage from `en` to  `en-US`?

Reply:
Yes. Then front end falls back to use language `en` if `en-US` is missing. And the same for using `zh` if `zh-CN` is missing.

Reply:
Change the i18n folder name and set default to `zh-CN` and `en-US`

Reply:
[explore api](https://github.com/awslabs/clickstream-analytics-on-aws/blob/main/src/control-plane/backend/lambda/api/test/api/reporting.test.ts#L232) use this language save in localStorage.